views/index.hbs

```diff
@@ -219,6 +229,14 @@
             </div>
           </form>
           <script>
+              function handleIdTypeClick(idType) {
+                if (idType.value === "email") {
+                  document.getElementById('id_input').placeholder = "Enter your email address";
```

Comment:
probably need to prepopulate both and hide one to handle locale?

Reply:
there is one input, simpler